core/src/main/java/org/arquillian/cube/impl/client/CubeLifecycleController.java

```diff
@@ -31,7 +31,7 @@ public void changeToPreRunning(@Observes PreRunningCube event, CubeRegistry regi
         validateAndGet(registry, event.getCubeId()).changeToPreRunning();
     }
 
-    private Cube<?> validateAndGet(CubeRegistry registry, String cubeId) {
+    public static Cube<?> validateAndGet(CubeRegistry registry, String cubeId) {
```

Comment:
Why static? Maybe we should extract it to a validator class and inject in all the components where needed?